app/javascript/pages/Calendar/index.js

```diff
@@ -192,6 +219,29 @@ const withData = compose(
               }),
             },
           },
+          update: (cache, { data: { destroySignup: { event: eventChange } } }) => {
```

Comment:
Could we reuse the callback above? Looks like only the parameter changes?

Reply:
Good point, I felt like I forgot something 😅